driver/ppm_events.h

```diff
@@ -81,7 +81,9 @@ struct event_filler_arguments {
 #endif	
 };
 
+#ifndef WDIG_CBACK
 extern const struct ppm_event_entry g_ppm_events[];
+#endif
```

Comment:
Is WDIG_CBACK defined anywhere?

Reply:
in the wdig code, which I have not open sourced yet